server/routes/recentlyAdded.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express');
+
+const createRecentlyAddedContentRouter = ({ cmsService }) => {
+  const router = express.Router();
+
+  const body = req => {
```

Comment:
Seems an odd name does `body`; its always assigned to related Content, so maybe `getRelatedContent`?

Reply:
I have updated the 'body' to be named, 'getMostRecentContent' as this is provides better context of what's actually happening. In addition, I have updated 'relatedContent' to be named 'mostRecentContent' for the same reason this is then assigned to the 'relatedContent' field in the data JSON that's passed into the view. 

In the context of this route 'relatedContent' doesn't make sense as this content isn't related to anything. However, the collections view that is used by both this route and again in tags.js which refers to 'relatedContent'. Ideally we would rename 'relatedContent' in both tags.js and the view too, to something more generic, like 'content' to avoid confusion in routes that use the view but aren't displaying 'related content'.

Reply:
Agreed, do you want to raise a _tidy up_ ticket for it?